.circleci/config.yml

```diff
@@ -90,52 +111,58 @@ jobs:
                 name: Integration Tests - Usage Apps Rendering
                 command: |
                     . venv/bin/activate
-                    pytest --nopercyfinalize --headless tests/test_usage.py
+                    python -m pytest --nopercyfinalize --headless tests/test_usage.py
```

Comment:
Curious why this change was necessary? `pytest` was finding the one in the base environment rather than the venv or something? We haven't needed to add `python -m` in front of other similar commands

Reply:
nonblocking, but I'm still curious about this change 😸

Reply:
It is actually not needed. I added it because at some point circleci was failing to find `pytest` and this seemed to solve it. I just removed it.